types/react-native/v0.65/index.d.ts

```diff
@@ -360,7 +360,7 @@ export interface HostComponent<P> extends Pick<React.ComponentClass<P>, Exclude<
 
 // see react-jsx.d.ts
 export function createElement<P>(
-    type: React.ReactType,
+    type: React.ElementType,
```

Comment:
tried to remove this change from this stack in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/59519 but CI is failing because it did not recognize all packages it needs to installed. tracking issue in https://discord.com/channels/508357248330760243/757992230077333655/958099454848815114